### PR TITLE
Prepare Repo for DocWarden Updates 

### PR DESCRIPTION
**Pull request by chidozieononiwu**
In preparation for updates to DocWarden tool to enforce changelog convention as defined 
 in [ChangeLog Guidiance](https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#changelog-guidance)
- Rename changlogs to All Caps
- Update latest entry in each changelog to follow the convention
- Update docsettings.yml
- Update Packages.md

Doc Warden will now function thus
```
Example Changelog
------------------------------- No latest Entry will block PR and CI
# 12.1.0-preview.1-------- Latest Entry Present will not block PR or CI
------------------------------- Empty Release notes will not block PR but block release
# 12.0.0-preview.4
```

### CODE REVIEW

**Comment by weshaggard** [comment]
>
```
Example Changelog
------------------------------- No latest Entry will block PR and CI
## 12.1.0-preview.1-------- Latest Entry Present will not block PR or CI
------------------------------- Empty Release notes will not block PR but block release
## 12.0.0-preview.4
```

@chidozieononiwu this seems to conflict with the guidance at https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#changelog-guidance we should try and make the guidance and the tool enforcement match :)

**Comment by chidozieononiwu** [comment]
> Example Changelog
> ------------------------------- No latest Entry will block PR and CI
> ## 12.1.0-preview.1-------- Latest Entry Present will not block PR or CI
> ------------------------------- Empty Release notes will not block PR but block release
> ## 12.0.0-preview.4
> 
> @chidozieononiwu this seems to conflict with the guidance at https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#changelog-guidance we should try and make the guidance and the tool enforcement match :)

I have fixed this now.

File: sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md [hunk 1]
```diff
@@ -26,7 +29,7 @@ This package's
 and 
 [samples](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration) 
 
-## Version 1.0.0-preview.5 (2019-10-11)
+## 1.0.0-preview.5
```
Comment (JonathanGiles) [inline sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md]: I note that further down this PR you include the dates, whereas here you remove them. Is there an inconsistency here?

Comment (chidozieononiwu) [reply sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md]: I have updated that file.

File: sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # Release History
+
+## 5.0.0-beta.7
```
Comment (JonathanGiles) [inline sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md]: Should this be (Unreleased)?

File: sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # Release History
-## 4.0.1 (2019-12-04)
 
+## 4.1.0-beta.1
```
Comment (JonathanGiles) [inline sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md]: Should this be (Unreleased)?

File: sdk/core/azure-core/CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,6 @@
-# Change Log azure-core
+# Release History
+
+## 1.2.0-beta.1
```
Comment (JonathanGiles) [inline sdk/core/azure-core/CHANGELOG.md]: Should this be (Unreleased)?

File: sdk/batch/microsoft-azure-batch/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,8 @@
-# Azure Batch SDK for Java release notes
-## Changes in 7.0.0
+# Release History
+
+## 7.1.0-beta.1
```
Comment (JonathanGiles) [inline sdk/batch/microsoft-azure-batch/CHANGELOG.md]: Should this be (Unreleased)?

File: sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md [hunk 6]
```diff
@@ -1,15 +1,18 @@
-# Change Log azure-data-appconfiguration
-## Version 1.0.0-preview.7 (2019-11-26)
+# Release History
+
+## 1.0.0-beta.8 (Unreleased)
+
+## 1.0.0-preview.7
```
Comment (JonathanGiles) [inline sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md]:
```suggestion
## 1.0.0-preview.7 (2019-11-26)
```

Comment (JonathanGiles) [reply sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md]: You merged to the repo before merging my suggested change. Perhaps this was missed?

Comment (chidozieononiwu) [reply sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md]: Opps, sorry. I'll fix that in another Pr

Comment (chidozieononiwu) [reply sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md]: Fixed here https://github.com/Azure/azure-sdk-for-java/pull/6932/files

**Pull request merged**